lms/static/sass/discussion/views/_create-edit-post.scss

```diff
@@ -11,6 +11,11 @@
   padding: $baseline;
   max-width: 1180px;
 
+  .post-errors {
+    padding: 0 !important;
```

Comment:
Uggh, those rules directly on elements are terrible! I think what you have makes sense then. If you want to move them to shame that would be great, or you could just leave them here.